src/creatures/players/player.cpp

```diff
@@ -1162,56 +1141,39 @@ ReturnValue Player::rewardChestCollect(const Container* fromCorpse /* = nullptr*
 	auto rewardCount = rewardItemsVector.size();
 	bool fallbackConsumed = false;
 	Item* fallbackItem = getInventoryItem(CONST_SLOT_BACKPACK);
-	int32_t movedMoney = 0;
+	int32_t movedRewardMoney = 0;
 	for (auto item : rewardItemsVector) {
```

Comment:
My first review so I'm not sure if this is the way to do it but I think adding the following would be good so it respects the skipped loot list.

		// Check if the item is in the ignore list
		bool isIgnored = isQuickLootListedItem(item);
		if (isIgnored && quickLootFilter == QUICKLOOTFILTER_SKIPPEDLOOT) {
			continue;
		}

Reply:
> My first review so I'm not sure if this is the way to do it but I think adding the following would be good so it respects the skipped loot list.
> 
> ```
> 	// Check if the item is in the ignore list
> 	bool isIgnored = isQuickLootListedItem(item);
> 	if (isIgnored && quickLootFilter == QUICKLOOTFILTER_SKIPPEDLOOT) {
> 		continue;
> 	}
> ```

It's a valid point, but I'm not sure if the ignore list is valid in the context of reward collect, as the idea is likely to collect all loot from a boss regardless of the list, someone playing the global would need to confirm the correct behaviour for to be sure.